### ISSUE TITLE
Help people find new ProofMode issue tracker on Gitlab!

### ISSUE THREAD

**Issue by nealmcb** [guardianproject/proofmode-android#115]
I see that the main git repo moved to Gitlab, apparently in late September 2023: https://gitlab.com/guardianproject/proofmode/proofmode-android

I don't see an issue tracker on gitlab.
**Ahh, after googling, I now see that you have to dig down on the left sidebar to Plan > Issues** Sigh.

Here is a direct link: https://gitlab.com/guardianproject/proofmode/proofmode-android/-/issues

Have all the old issues been moved over? There seem to be some discrepancies, and I guess the tracking of Milestone updates got lost.

Please update the README here, or somehow make this all more obvious than this issue.

And I dare say that at some point the github repo should become a regularly-updated mirror of the gitlab repo.

**Comment by n8fr8** [comment]
Yes, thanks for all this feedback, and sorry for the lag in pointing people to the new spot. Will be updating here some more and hopefully get some help closing and addressing any still open issues here.

**Comment by nealmcb** [comment]
Glad to see you back! Thanks. 
Can I suggest leaving this open until the readme here is updated? Otherwise you now have to dig in closed issues to see that everything is not as it seems here.
